### ISSUE TITLE
Finalize variable naming conventions(cases) for code uniformity

### ISSUE THREAD

**Issue by abhishekmsingh** [abhiandthetruth/wardrobe-api#22]
We use python and we need a case for that. Its snake case for variables and title case for classes right?

**Comment by Shreya-7** [comment]
yes, here is the pep8 doc https://peps.python.org/pep-0008/#naming-conventions

**Comment by Shreya-7** [comment]
As for enforcing, we will need flake8 + pep8-naming to throw errors/warnings and black as formatter. 
Any auto reformatting should not be risked before tests
